qmerge.c

```diff
@@ -919,12 +919,17 @@ pkg_merge(int level, const depend_atom *atom, const struct pkg_t *pkg)
 	 * name suggests, bug #660508, usage of BINPKG_COMPRESS,
 	 * due to the minimal nature of where we run, we cannot rely on file
 	 * or GNU tar, so have to do some laymans MAGIC hunting ourselves */
-	compr = "j";  /* default: bzip2 */
+	compr = "I brotli"; /* default: brotli; has no magic header */
 	{
-		/* bz2: 2-byte: 'B' 'Z'                  at byte 0
-		 * gz:  4-byte:  1f  8b                  at byte 0
+		/* bz2: 3-byte: 'B' 'Z' 'h'              at byte 0
```

Comment:
so, we only detect huffman encoding? or is that the only format in use?

Reply:
https://en.wikipedia.org/wiki/Bzip2#File_format
'h' for Bzip2, '0' for Bzip1 (obsolete).
bzip2 always writes 'h'.